vpc-flow-logs.tf

```diff
@@ -10,6 +10,7 @@ resource "aws_cloudwatch_log_group" "vpc_flow_logs" {
   count             = var.vpc_flow_logs ? 1 : 0
   name              = "/aws/vpc/${var.name}-VPC/flow-logs"
   retention_in_days = var.vpc_flow_logs_retention
+  kms_key_id        = length(var.kms_key_arn) > 0 ? var.kms_key_arn : ""
 
```

Comment:
If we don't have a default value for the variable, the logic does not work, as you always need to pass a value for kms_key_arn it will never be zero, unless the person set is as an empty string.

Reply:
That's a good point. 

I'll improve this solution.